sxg_rs/src/headers.rs

```diff
@@ -206,37 +214,66 @@ fn validate_sxg_request_header(accept: &str) -> Result<(), String> {
 
 #[cfg(test)]
 mod tests {
+    use std::fmt::Debug;
     use super::*;
+
+    fn header_fields(pairs: Vec<(&str, &str)>) -> HeaderFields {
+        pairs.into_iter().map(|(k,v)| (k.to_string(), v.to_string())).collect()
+    }
+    fn headers(pairs: Vec<(&str, &str)>) -> Headers {
+        Headers::new(header_fields(pairs))
+    }
+    fn assert_contains<T, U: 'static>(result: Result<T, U>, value: T) where T: Debug + PartialEq, U: Debug + Send {
+      assert!(result.is_ok(), result.unwrap_err());
```

Comment:
The `is_ok` is not needed to be checked. If `result` contains an error, the `unwrap` in the next line will throw the error by panicking.

Reply:
Ah, good point; the failure message is just as good. Okay, that solves the first issue for me.